src/molecules/formfields/CheckboxList.js

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+import styles from './formfields.module.scss';
+import classnames from 'classnames';
+
+function CheckboxList ( props ) {
+  const {
+    label,
+    focused,
+    data,
+    handleClick
+  } = props;
+
+  return (
+    <div className={ classnames(styles.checkboxList, { [styles.focused]: focused })}>
+      <label className={ styles.label }>{ label }</label>
+      <div className={ styles.content }>
+        {data.map((item, idx) => {
+          return (
+            <label className={ styles.checkbox } key={ idx } onClick={ () => handleClick( item.id ) }>
```

Comment:
hmm yeah I was trying to think for a while how best to do this. The problem with passing back the item is that there's no guarantee of anything in the item being a unique identifier, although I guess that would be the responsibility of the end user to make sure there is something unique in the data... I will change to `item`, and I'm sure we'll get good feedback as devs start to implement stuff using the library